api/src/courses/test_courses.py

```diff
@@ -688,7 +688,7 @@ def test_create_course_offering(auth_client):
 
 def test_read_all_course_offerings(auth_client):
     # GIVEN existing (active and inactive) course offerings
-    create_multiple_courses(auth_client.sqla, 1)
+    create_multiple_(auth_client.sqla, 1)
```

Comment:
This looks like an unintentional change.

Reply:
will be fixed